bumble/helpers.py

```diff
@@ -232,3 +233,26 @@ def __init__(
         )
         self.host_to_controller_analyzer.peer = self.controller_to_host_analyzer
         self.controller_to_host_analyzer.peer = self.host_to_controller_analyzer
+
+
+def generate_irk() -> bytes:
+    return crypto.r()
+
+
+def gen_rpa_with_irk(irk: bytes) -> bytes:
```

Comment:
suggest: naming this `generare_rpa_with_irk` to be consistent with `generate_irk`.

Reply:
Done